place/models.py

```diff
@@ -1,59 +1,64 @@
 from django.db import models
 
 class Category(models.Model):
-    name = models.CharField(max_length=50)
+    name                        = models.CharField(max_length=50, unique=True)
```

Comment:
```suggestion
    name = models.CharField(max_length=50, unique=True)
```
이렇게 띄우시진 않으셔도 됩니다. 코드 간격은 같은 클래스 내부나 같은 함수 내부에서만 지켜주시면 됩니다 👍

Reply:
넵 반영하겠습니다